omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -315,12 +315,12 @@
                     }
 
                     if (!patient_regimen.includes(regimen_encounter_page_value)) {
-                        jq().toastmessage('showNoticeToast', "Regimen: "+regimen_encounter_page_text+" is not in patient regimens on ART Summary page");
+                        jq().toastmessage('showNoticeToast', "Regimen: " + regimen_encounter_page_text + " is not in patient regimens on ART Summary page");
```

Comment:
@ssmusoke  this will be addressed in another card not the same card. This comment should not be for this PR.